salesagility/SuiteCRM
# Fixed #6611 -

This is a fix for #6611 
## Description
This now allows you to send an email from List View with the variable for example "$contact_first_name" to be populated and filled out as it is in Detail View.  

## How To Test This
1. Go to accounts
2. Go to the contact sub panel
3. Select an Email
4. Add the Email Template with the Variable in it.
5. Send the Email to check if the variable has changed

## Types of changes

- [X ] Bug fix (non-breaking change which fixes an issue)

### Final checklist
